doc/source/rllib/key-concepts.rst

```diff
@@ -164,7 +164,7 @@ RLModules
     The following is a quick overview of **RLlib RLModules**.
     See :ref:`here for a detailed description of the RLModule class <rlmodule-guide>`.
 
-`RLModules <rllib-rlmodule.html>`__ are deep-learning framework-specific neural network containers.
+`RLModules <rl-modules.html>`__ are deep-learning framework-specific neural network containers.
```

Comment:
Maybe better "wrappers" to not confuse users with Docker containers.

Reply:
done